src/types/common/main.d.ts

```diff
@@ -21,11 +21,7 @@ export interface CustomizeBobaModalProps {
   onHide: () => void;
 }
 
-export interface ToppingProps {
-  name: string;
-}
-
-export interface MilkProps {
+export interface ModalOptionProps {
```

Comment:
remove unused interface.
Too simple of interface as single string type, that needs to be inside `main.d.ts` Just define directly inside props if needed